fixes
  gimple-fold.h:72:32: error: use of enum 'gsi_iterator_update' without previous declaration
for some reason others don't seem to have this issue, but it is trivial
https://lore.kernel.org/lkml/DB6P189MB05681E9F4785DF2758B9875B9CA49@DB6P189MB0568.EURP189.PROD.OUTLOOK.COM/t/
--
--- a/scripts/gcc-plugins/gcc-common.h
+++ b/scripts/gcc-plugins/gcc-common.h
@@ -71,6 +71,8 @@
 #include "varasm.h"
 #include "stor-layout.h"
 #include "internal-fn.h"
+#include "gimple.h"
+#include "gimple-iterator.h"
 #include "gimple-expr.h"
 #include "gimple-fold.h"
 #include "context.h"
